src/lib/components/FormFeedback.js

```diff
@@ -193,30 +193,33 @@ class DisconnectedFormFeedback extends Component {
     //          e.g., {metadata: {creators: ,,,}} => {"metadata.creators": ...}
     // For now, only for metadata, files and access.embargo
     const metadata = errors.metadata || {};
-    // eslint-disable-next-line camelcase
```

Comment:
@zzacharo Not sure I fully understand the need for `_` instead of camel case. It seems the decision was consious since the linter is disabled. However, I do not find why. I've tried to understand if its on step 1 to calculate the field of the error but it's actually on the value. 🤔

Reply:
I just followed the convention that was there before. Well changed 👏